src/Collection/CollectionInterface.php

```diff
@@ -542,23 +539,23 @@ public function shuffle(): CollectionInterface;
      * Returns a new collection with maximum $size random elements
      * from this collection
      *
-     * @param int $size the maximum number of elements to randomly
+     * @param int $length the maximum number of elements to randomly
      * take from this collection
      * @return self
      */
-    public function sample(int $size = 10): CollectionInterface;
+    public function sample(int $length = 10): CollectionInterface;
 
     /**
      * Returns a new collection with maximum $size elements in the internal
      * order this collection was created. If a second parameter is passed, it
      * will determine from what position to start taking elements.
      *
-     * @param int $size the maximum number of elements to take from
+     * @param int $length the maximum number of elements to take from
      * this collection
-     * @param int $from A positional offset from where to take the elements
+     * @param int $offset A positional offset from where to take the elements
      * @return self
      */
-    public function take(int $size = 1, int $from = 0): CollectionInterface;
+    public function take(int $length = 1, int $offset = 0): CollectionInterface;
```

Comment:
Wouldn't `$count` be more appropriate?

Reply:
I was following the naming style from array_splice - https://www.php.net/manual/en/function.array-splice.php.

I used $length everywhere to see if it worked.

Reply:
I am not convinced it's the best name but NVM, being consistent with PHP's naming is good enough.